src/couch_jobs/src/couch_jobs_fdb.erl

```diff
@@ -355,6 +356,13 @@ get_activity_vs_and_watch(#{jtx := true} = JTx, Type) ->
     end.
 
 
+get_active_jobs(Type) ->
+    fabric2_fdb:transactional(fun(Tx) ->
+        JTx = init_jtx(Tx),                                                        
+        get_active_since(JTx, Type, {versionstamp, 0, 0})
+    end).
+
+
```

Comment:
We are calling get_active_since directly from couch_jobs module so we don't need this function anymore.